### ISSUE TITLE
Re-implement all NH features into CBRAIN

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#1244]
The NeuroHub alternative portal developed in 2020 has some features that are not available on the CBRAIN side.

Let's make sure the CBRAIN interface now provides access to all the same features.

- [ ] Allow users to create their own UserKeyFlatDirDataProvider (including auto-register feature?) (PR)
- [ ] Allow project owners to manage project a list of editors (PR)
- [x] ORCID ID auth support
- [ ] Work Project can have custom licenses that must be accepted by new members ( MontrealSergiy - I will try to work on that ) (PR)
- [ ] invite user to your project by email (in cbrain direct user selection) (PR)
- [ ] user can make project public (PR)
- [x] Specialized API hooks for LORIS integrations
- [x] user to user messages

See also the Release notes for CBRAIN version 6.0.0

**Comment by MontrealSergiy** [comment]
Hi, are following features deemed low priority 

- invite to project by email (in cbrain direct user selection)
- user can make project public
- Specialized API hooks for LORIS integrations
- user2user messages

are not needed?

**Comment by prioux** [comment]
No, all features should be supported. I just didn't list them all in the initial description. Add them to the list.

**Comment by MontrealSergiy** [comment]
@prioux should I work on some of those?

**Comment by bryancaron** [comment]
As discussed with Pierre, the 'user-to-user messaging' can be left out of the functionalities to transfer from NeuroHub to CBRAIN.

**Comment by MontrealSergiy** [comment]
looks like user dp is still in progress
